packages/smooth_app/lib/pages/scan/search_history_view.dart

```diff
@@ -38,8 +38,10 @@ class _SearchHistoryViewState extends State<SearchHistoryView> {
   @override
   Widget build(BuildContext context) {
     return ListTileTheme(
-      data: const ListTileThemeData(
-        titleTextStyle: TextStyle(fontSize: 20.0),
+      data: ListTileThemeData(
+        iconColor: Theme.of(context).colorScheme.primary,
```

Comment:
`primary`? That's not the purpose of a primary color to be the text color in a ListTile, is that?
Please find something more relevant, like `onBackground`.
cf. https://m1.material.io/style/color.html#color-color-tool